app/features/quiz/QuizSession/reducer.js

```diff
@@ -43,7 +43,8 @@ const toggleWrapUp = (state) => {
 
   if (active) {
     queue = getWrapUpItems(state);
-    count = queue.length;
+    // wrapUp count should be 1 less than queue length
+    count = queue.length - 1;
```

Comment:
Changed this to ensure that the wrapUp count remains consistently 1 less than the queue length (otherwise we sometimes never trigger the wrapUp exit, but haven't queue up another review to get us there)